roles/node-app/tasks/main.yml

```diff
@@ -30,7 +30,7 @@
   file: path={{ node_app_path }} state=directory mode=0755 owner={{ node_app_user }} group={{ node_app_group }}
 
 - name: "{{ node_app_name }} | Pull sources from the repository"
-  git: repo={{ node_app_git_repo }} dest={{ node_app_path }} force=yes version=master
+  git: repo={{ node_app_git_repo }} dest={{ node_app_path }} force=yes version={{ node_app_git_branch }}
```

Comment:
`node_app_git_branch` variable probably should have `master` as default value e.g. in `roles/node-app/defaults/main.yml`. Otherwise every use of `node-app` role except the provided `signalk` role will break after this change until a value for `node_app_git_branch` is explicitly provided.

Reply:
Fair enough, I'll revert the default back to master.

Reply:
I mean, add it as you suggested. :-D